controller/src/main/java/io/pravega/controller/server/rpc/auth/PravegaAuthManager.java

```diff
@@ -42,91 +41,52 @@ public PravegaAuthManager(GRPCServerConfig serverConfig) {
     private AuthHandler getHandler(String handlerName) throws AuthenticationException {
         AuthHandler retVal;
         synchronized (this) {
-            retVal = handlerMap.get(handlerName);
-        }
-        if (retVal == null) {
+                retVal = handlerMap.get(handlerName);
```

Comment:
Indentation seems incorrect in this block.

Reply:
Fixed.